.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -14,7 +14,7 @@ body:
   - type: checkboxes
     attributes:
       label: Verification
-      description: Please verify that you've followed these steps.
+      description: Please verify that you've followed these steps. If you cannot truthfully check these boxes, open a discussion at https://github.com/orgs/Homebrew/discussions instead.
```

Comment:
```suggestion
      description: Please verify that you've followed these steps. If you cannot, open a discussion at https://github.com/orgs/Homebrew/discussions instead.
```

This would be more direct but maybe we want to be really explicit here.

Reply:
Would rather leave as-is for now 🙇🏻